File: Scripts/newTask/json/ruby/json.rb
require 'json'

for i in 0..10000000
ruby_obj = JSON.parse('{"blue": [1, 2], "ocean": "water"}')
#puts ruby_obj

ruby_obj["ocean"] = { "water" => ["fishy", "salty"] }
#puts JSON.generate(ruby_obj)
#puts JSON.pretty_generate(ruby_obj)
JSON.generate(ruby_obj)
JSON.pretty_generate(ruby_obj)

end

File: Scripts/newTask/json/c++/json.cpp
#include <iostream>
#include <iomanip> // std::setw
#include <sstream>
#include <cassert>

#include "json.hpp"

using json = nlohmann::json;

int main( int argc, char* argv[] )
{
	for (int i = 0; i < 10000000; ++i) {
        	std::string const expected = R"delim123({"foo": 1, "bar": {"number": "10", "fruit": "apple"}})delim123";
		json const jexpected = json::parse( expected );

		json newJson = {{"blue"},{{"number_1","1"}, {"numbe_2","2"}}, {"ocean", "water"}};
		std::stringstream newJson_stream;
    		newJson_stream << std::setw(4) << newJson; 
		
	}
    return 0;
}
